wdl-lint/src/rules/import_sort.rs

```diff
@@ -64,6 +64,10 @@ impl Rule for ImportSortRule {
     fn exceptable_nodes(&self) -> Option<&'static [SyntaxKind]> {
         Some(&[SyntaxKind::VersionStatementNode])
     }
+
+    fn related_rules(&self) -> &[&'static str] {
+        &["InputSorting", "SectionOrdering"]
```

Comment:
```suggestion
        &[]
```
if we removed it from this connection from the other rule we should be consistent